ros2pkg/ros2pkg/package_type/__init__.py

```diff
@@ -0,0 +1,83 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Needs copyright line to pass linter. Please run the tests of the package.

Reply:
Fixed in latest update.